update.go

```diff
@@ -254,6 +259,14 @@ other options are ignored.
 		config.Cgroups.Resources.MemorySwap = *r.Memory.Swap
 		config.Cgroups.Resources.PidsLimit = r.Pids.Limit
 
+		if val := context.String("l3-cache-schema"); val != "" {
+			// Avoid silent failure in container.Set()
+			if !intelrdt.IsEnabled() || config.IntelRdt == nil {
+				return fmt.Errorf("l3 cache schema is not enabled or not specified in config, update value '%s' is discarded", val)
```

Comment:
This error should be shorter, just remove the last bit `, update value '%s' is discarded`.

Reply:
@cyphar 

Thank you for review. This sounds fine to me.